src/Dofs/DofHandler.jl

```diff
@@ -773,8 +757,33 @@ function sortface_fast(face::Tuple{Int,Int,Int,Int})
 end
 
 
-sortface(face::Tuple{Int}) = face, nothing
-sortface_fast(face::Tuple{Int}) = face
+"""
+    sortfacet(facet::NTuple{N, Int})
+
+Returns the unique representation of the `facet` by sorting its node indices. 
+Dispatches on `sortedges` or `sortfaces` depending on `N`
+"""
+function sortfacet end 
+
+"""
+    sortfacet_fast(facet::NTuple{N, Int})
+
+Returns the unique representation of the `facet` by sorting its node indices. 
+Dispatches on `sortedges_fast` or `sortfaces_fast` depending on `N`
+"""
+function sortfacet_fast end 
+
+# Vertex
+sortfacet(facet::Tuple{Int}) = facet
+sortfacet_fast(facet::Tuple{Int}) = (facet, nothing)
```

Comment:
was this covered bevor in tests when it was named `sortface_fast` ? I'm just wondering about the codecov complains

Reply:
Good spot, this is wrong. 
Functions definitions should be reversed.

Reply:
I guess, because `sortface` and `sortedge` is used in the dof distribution.
The only place we use `sortfacet` is https://github.com/Ferrite-FEM/Ferrite.jl/blob/f8c67406bffa90afcaa92817fbdea7d161d11850/src/Dofs/ConstraintHandler.jl#L1355-L1368

Reply:
Looks like only `sortfacet_fast` will be required actually...